content/02.introduction.md

```diff
@@ -24,69 +24,59 @@ Further, we aim to amplify the true signal out of the vast noise produced by tho
+The conditions within the phagosome are then modulated to destroy the pathogen through fusion of the phagosome with a lysosome or granule to form a phagolysosome.
+This allows the cell to break the pathogen into its component peptides. 
+Monocytes, which include both dendritic cells and macrophages, are both phagocytic white blood cells and professional antigen-presenting cells, which meaning that they can then present the resulting viral peptides on the cell surface using Major Histocompatability Complex II (MHC-II) proteins.
+However, research in related viruses suggests that SARS-CoV-2 may possess some mechanisms that facilitate evasion of the antibody-mediated response.
+_In vitro_ analysis of nidoviruses including SARS-CoV suggests that these viruses can induce the development of double-membrane vesicles, preventing the dsRNA signature from being detected by the host immune system [@doi:10.1128/jvi.02501-05].
+This protective envelope can therefore insulate these coronaviruses from the innate immune response's detection mechanism [@doi:10.1016/j.jpha.2020.03.001].
+Presentation of the antigen is important because this step allows it to be detected by B-lymphocytes, also called B-cells, which are a type of white blood cell with antibodies bound to the membrane.
+Individual B-cells have unique membrane-bound antibodies.
+A B-cell is able to bind to an antigen only if its antibody is a match.
+For example, toll-like receptor 3 (TLR3) can detect double-stranded RNA (dsRNA) present during viral replication, allowing it to recognize a viral threat.
+The binding of a pattern recognition receptor (PRR), such as a TLR, to the antigen allows the B-cell to engulf it the virus via endocytocis.
+Once the antigen has been processed and presented on the cell surface, helper T-cells can bind to the MHC-II/antigen complex.
+This second binding step catalyzes the release of cytokines as part of the inflammatory response, as well as the proliferation of active B-cells (plasma cells) capable of producing the antibodies that respond to the antigen.
+Coronaviruses such as SARS-CoV are also able to evade the humoral immune response through other mechanisms, such as inhibiting certain cytokine pathways or down-regulating antigen presentation by the cells [@doi:10.1128/jvi.02501-05].
+Additionally, the cell will produce memory B-cells that can retain information about how to respond to future immune challenges by this pathogen.
+Understanding the humoral response to SARS-CoV-2 is important because this component of the immune response works to reduce the severity of infection in the short term and is also the mechanism through which the body can respond to reexposure.
+
+Research on other coronaviruses provides some evidence of how the innate immune response may proceed in the presence of SARS-CoV-2.
```

Comment:
what follows is acquired immune response so you can just say "...how the immune response may proceed..." / remove the specification of "innate" (you might consider remove "innate" from the whole subsection here & just find a subheading which says that you're looking at specific immune system features related to SARS-CoV-2)